homeassistant/components/harman_kardon_avr/media_player.py

```diff
@@ -94,11 +140,13 @@ def source(self):
     @property
     def source_list(self):
         """Available sources."""
-        return self._source_list
+        return self._sources
```

Comment:
We should not allow overriding the sources via the config. What is wrong with the options that come from the API?